test/liquidity-protection/LiquidityProtection.js

```diff
@@ -3802,6 +3426,536 @@ describe('LiquidityProtection', () => {
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * |               | Liquidity Provider | V3 Bancor Vault | Liquidity Protection Wallet |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | Gov Token     |          I         |        I        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | XXX Token     |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | YYY Token     |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | ETH "Token"   |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | Network Token |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | Pool Token    |          I         |        I        |              D              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * The ones marked I are expected to be identical before and after each migration request
+             * The ones marked D are expected to be different before and after each migration request
+             *
+             * The state of the system also includes the following counters:
+             * - System balance in the XXX/BNT pool
+             * - System balance in the YYY/BNT pool
+             * - System balance in the ETH/BNT pool
+             * - Total amount of XXX/BNT pool tokens
+             * - Total amount of YYY/BNT pool tokens
+             * - Total amount of ETH/BNT pool tokens
+             * - Total amount of XXX in the XXX/BNT pool
+             * - Total amount of BNT in the XXX/BNT pool
+             * - Total amount of YYY in the YYY/BNT pool
+             * - Total amount of BNT in the YYY/BNT pool
+             * - Total amount of ETH in the ETH/BNT pool
+             * - Total amount of BNT in the ETH/BNT pool
+             * - Per provider amount of XXX in the XXX/BNT pool
+             * - Per provider amount of BNT in the XXX/BNT pool
+             * - Per provider amount of YYY in the YYY/BNT pool
+             * - Per provider amount of BNT in the YYY/BNT pool
+             * - Per provider amount of ETH in the ETH/BNT pool
+             * - Per provider amount of BNT in the ETH/BNT pool
+             */
+            describe('liquidity-migration tests', () => {
```

Comment:
There don't seem to be tests migrating multiple "migration units" in the same call

Reply:
The primary test (the first `it`) does.
Search for `migratePositions(migrationUnits)`.